docs/developers/node-tutorial.md

```diff
@@ -0,0 +1,551 @@
+# Getting and Sending Transactions with Celestia Node
+<!-- markdownlint-disable MD013 -->
+
+In this tutorial, we will cover how to use the Celestia Node API to submit and
+retrieve messages from the Data Availability Layer by their namespace ID.
+
+## Hardware Requirements
+
+You can find hardware requirements [here](../nodes/light-node#hardware-requirements).
+
+## Setting Up Dependencies
+
+You can follow the tutorial for setting up the dependencies [here](../developers/environment).
+
+## Celestia Node
+
+## Install Celestia Node
+
+You can follow the tutorial for building Celestia Node [here](../developers/celestia-node)
+
+### Instantiate Celestia Light Node
+
+Now, let's instantiate a Celestia Light node:
+
+> Note: RPC Endpoints are exposed in all Celestia Node types
+  such as Light, Bridge and Full Nodes.
+
+```sh
+./build/celestia light init
```

Comment:
in some cases, we do `./build/celestia`, in others we just do `celestia`

lets keep it `celestia` bc we expect users to do make install